### PR TITLE
cli for vogue load samples and vogue load flowcells

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds CLI to run vogue load samples and vogue load flowcells

**How to test**:

On hasta
cd /home/proj/stage/servers/resources/hasta.scilifelab.se

1. bash update-servers-stage.sh master

2. bash update-cg-stage.sh vogue_api

3. bash update-vogue-stage.sh master

4.  run: 
usestage
export VOGUE_MONGO_URI=mongodb://localhost:27030
export VOGUE_MONGO_DBNAME=vogue-stage 
export VOGUE_FLASK_DEBUG=1
export VOGUE_SECRET_KEY=hej

- cg upload vogue samples -d 1

- cg upload vogue flowcells -d 1

**Expected test outcome**:
Both upload commands should run without errors and tell weter samples/flowcells were updated.

**Review:**
- [x] code approved by @patrikgrenfeldt 
- [x] tests executed by @mayabrandi 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by mayabrandi** [comment]
Testing:
Installed on stage according to above.

Loading flowcells worked fine:

Skärmavbild 2020-02-25 kl  10 01 40


Loading samples took a while

File: cg/apps/vogue.py [hunk 1]
```diff
@@ -38,3 +38,23 @@ def load_apptags(self, apptag_list: list):
...
+
+        load_call = ["load", "sample", "-d", days]
+        self.process.run_command(load_call)
+
+        # Execute command and print its stdout+stderr as it executes
+        for line in self.process.stderr_lines():
+            LOG.info("vogue output: %s", line)
+
+    def load_flowcells(self, days):
+        """Running vogue load samples."""
```
Comment (patrikgrenfeldt) [inline cg/apps/vogue.py]: flowcells

**Comment by patrikgrenfeldt** [comment]
code looks good, had  one comment. Could you motivate the "version" you are suggesting?

File: cg/apps/vogue.py (hunk 1 above)
Comment (mayabrandi) [inline cg/apps/vogue.py]:
```suggestion
        """Running vogue load flowcells."""
```

**Comment by mayabrandi** [comment]
its a minor bump version because Im adding functionality that will not affect anything existing.

**Comment by mayabrandi** [comment]
Ready for deploy? @patrikgrenfeldt

**Comment by patrikgrenfeldt** [comment]
deployed on hasta

**Pull request merged**